docs/build/security-docs/threat-modeling/STRIDE-template.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 30
+sidebar_label: STRIDE Threat Model Template
+title: STRIDE Threat Model Template
+---
+
+## What are we working on?
+
+_**Text**: high level description of the project and what is being designed_
+
+_At least one visual dataflow diagram that shows process flows between different entities in the system_
```

Comment:
I'm a little confused by the top of this page. The page is meant to be a template someone could use to write their own STRIDE assessment thing, right? Are these lines supposed to placeholders, or instructions? that might need to be cleared up a little bit.